Directory.Build.targets

```diff
@@ -29,7 +30,18 @@
       SourceFiles="@(MibcFiles)"
       DestinationFolder="$(ArtifactsDir)\mibc\%(MibcPackage.Identity)\"
       SkipUnchangedFiles="true" />
-
+  </Target>
+  
+  <!-- Copy all packages to the output (per architecture) - now for proto, sequentially, to avoid locking -->
+  <Target
+    Name="CopyPackageFilesToProto"
+    AfterTargets="CopyPackageFiles"
```

Comment:
Not sure how it'll help. I think lock happens in first one as well.

Reply:
Yeah it didn't, reverted that